engine/src/main/java/org/hibernate/validator/internal/cfg/context/ExecutableConstraintMappingContextImpl.java

```diff
@@ -125,6 +125,7 @@ public ConstrainedElement build(ConstraintHelper constraintHelper, TypeResolutio
 	private List<ConstrainedParameter> getParameters(ConstraintHelper constraintHelper, TypeResolutionHelper typeResolutionHelper,
 			ExecutableParameterNameProvider parameterNameProvider, ValueExtractorManager valueExtractorManager) {
 		List<ConstrainedParameter> constrainedParameters = newArrayList();
+		List<String> parameterNames = parameterNameProvider.getParameterNames( executable );
```

Comment:
`parameterNameProvider.getParameterNames( executable )` was called inside of the loop so I thought that extracting it out of it would be a bit faster.